src/components/SubscriptionStats.vue

```diff
@@ -1,10 +1,32 @@
 <template>
-  <div class="text-center">
-    <h3>Subscription</h3>
+  <div class="max-w-sm m-auto flex">
```

Comment:
The dashboard is made up of three separate components to keep things separated, minimal styling right now will be updating in next pr